recipes/minikube/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/env bash
+
+set -ex
+
+export GOPATH="${BUILD_PREFIX}/bin"
+export GOROOT="${BUILD_PREFIX}/go"
```

Comment:
I wonder if those should be set automatically for conda-forge.

Reply:
These are already set from a conda-forge. This is why I referred them to the consul-feedstock as an example.

Reply:
thanks for the feedback @ocefpaf and @sodre ... that makes sense we can address that on the next PR in the feedstock repo.